docs/quickstart/java_api_quickstart.md

```diff
@@ -94,7 +134,9 @@ data 1
 is 1
 ~~~
 
-The following code shows the WordCount implementation from the Quickstart which processes some text lines with two operators (FlatMap and Reduce), and prints the resulting words and counts to std-out.
+The following code shows the `WordCount` implementation from the
+Quickstart which processes some text lines with two operators (FlatMap
+and Reduce), and prints the resulting words and counts to std-out.
```

Comment:
The code does not show a `Reduce` but only `sum` which is executed as `Reduce`. Would be good to clarify this. Otherwise it might be confusing for readers.